Userland/Applications/PDFViewer/PDFViewerWidget.cpp

```diff
@@ -6,6 +6,13 @@
  */
```

Comment:
Commit Message nit: `The ThreeView has...` Did you mean TreeView?

Reply:
I did indeed. More importantly, the title reads "LibPDF: ..." when in fact the commit was all about PDFVIewrer, so I've changed that as well.